lib/matplotlib/rcsetup.py

```diff
@@ -215,24 +182,34 @@ def validate_dpi(s):
             ' could not convert "%s" to float' % (s, s))
 
 
-def validate_int(s):
-    """Convert s to int or raise."""
-    try:
-        return int(s)
-    except ValueError:
-        raise ValueError('Could not convert "%s" to int' % s)
+def _make_cls_validator(cls, *, allow_none=False):
```

Comment:
Not a big thing, and I know that these concepts are rather the same in python, but I would find it more clear to call this `_make_type_validator`.

Reply:
sure

Reply:
:smiley: